_inc/client/plans/single-product-backup.jsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { AccentedCard } from './accented-card';
+import Gridicon from 'components/gridicon';
+
+export function SingleProductBackup( props ) {
+	const { sitePlans } = props;
+
+	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
+
+	return (
+		<>
+			<h1 className="plans-section__header">Single Products</h1>
```

Comment:
The `h1` and `h2` headings seem a bit off here, as we already have an `h2` that seems to be more important hierarchically:

![image](https://user-images.githubusercontent.com/426388/68379918-1edc0000-014f-11ea-8c59-3d765c5c9564.png)